fairscale/utils/__init__.py

```diff
@@ -5,4 +5,10 @@
 
 from typing import List
 
+import torch
+
 __all__: List[str] = []
+
+
+def torch_version() -> List[int]:
```

Comment:
I think we already have a `torch_version` function in utils/testing.py. Maybe move it out to here?